site/content/_index.html

```diff
@@ -4,37 +4,50 @@
 description = "Agent Sandbox is a cloud native controller for sandboxes"
 +++
 
-{{< blocks/cover title="Agent Sandbox" image_anchor="top" height="full" color="black" >}}
+
+{{< blocks/cover subtitle="Agent Sandbox" title="Build Agents in a Safer Environment" color="black" >}}
```

Comment:
The font of the title, subtitle, and buttons still looks pretty big on my screen, especially comparing to the logo and other text: 
<img width="1591" height="861" alt="Image" src="https://github.com/user-attachments/assets/188a99a3-a741-4f3b-9d75-f82e355cf691" />

Also, the original design of the buttons look more polished, can we improve that in the new design as well?
<img width="807" height="553" alt="Image" src="https://github.com/user-attachments/assets/0f5c29cf-0d91-4fe6-9198-f0e67c695ff2" />

Reply:
Are title and subtitle reversed, or is that intentional?